tests/html5skin-test.js

```diff
@@ -521,6 +521,69 @@ describe('Controller', function() {
       expect(controller.state.controlBarVisible).toBe(true);
     });
 
+    describe('test customPlayPause', () => {
+      let spy;
+      let testEvent;
+      let testPlayerState;
+      const onTogglePlayPause = (event, playerState) => {
```

Comment:
I think it makes tests overcomplicated. You implement a method in the test and check it's implementation. Checking that it has been called using `toHasBeenCalledWith` would be enough

Reply:
`const onTogglePlayPause = jest.fn();`

Reply:
\+